sonar/src/main/java/com/rbc/sonar/Main.java

```diff
@@ -5,7 +5,9 @@
  */
 public class Main {
 
+    private static final String PASSWORD = "mypassword";
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Remove this hard-coded password. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9089/coding_rules#rule_key=squid%3AS2068)